build/_k-standard-goals.shade

```diff
@@ -187,6 +189,18 @@ default Quiet='${ false }'
     }
   }
 
+#nuget-verify target='test' if='File.Exists(PACKAGELIST_JSON_FILENAME)' description='Verify if all the packages are generated properly'
```

Comment:
Specifying it this way means that the target doesn't get defined at all. In which case, you don't have to have this check on every line